library/websphere-liberty

```diff
@@ -16,90 +16,18 @@ Tags: full, latest
 Directory: ga/latest/full
 File: Dockerfile.ubuntu.ibmjava8
 
+Tags: 20.0.0.3-kernel-java8-ibmjava
+Directory: ga/20.0.0.3/kernel
+File: Dockerfile.ubuntu.ibmjava8
+
+Tags: 20.0.0.3-full-java8-ibmjava
```

Comment:
This image is `FROM websphere-liberty:20.0.0.3-kernel`, but the only image tag is `20.0.0.3-kernel-java8-ibmjava` (this is the cause of the travis build failure).  Did you want to also have this more generic tag or does the Dockerfile need updating?

Reply:
The Dockerfile did need updating. That change has been made, and an update commit has been added to this PR to point to it.